langchain4j/src/main/java/dev/langchain4j/model/azure/AzureOpenAiStreamingLanguageModel.java

```diff
@@ -0,0 +1,97 @@
+package dev.langchain4j.model.azure;
+
+import dev.ai4j.openai4j.OpenAiClient;
+import dev.ai4j.openai4j.completion.CompletionRequest;
+import dev.langchain4j.model.StreamingResponseHandler;
+import dev.langchain4j.model.Tokenizer;
+import dev.langchain4j.model.language.StreamingLanguageModel;
+import dev.langchain4j.model.language.TokenCountEstimator;
+import dev.langchain4j.model.openai.OpenAiStreamingChatModel;
+import lombok.Builder;
+
+import java.net.Proxy;
+import java.time.Duration;
+
+import static dev.langchain4j.internal.ValidationUtils.ensureNotBlank;
+import static java.time.Duration.ofSeconds;
+
+/**
+ * Represents a connection to the OpenAI LLM with a completion interface, such as text-davinci-003.
```

Comment:
Represents a connection to the OpenAI LLM, hosted on Azure (like text-davinci-003).